packages/rum-core/src/domain/contexts/pageStateHistory.ts

```diff
@@ -31,6 +31,7 @@ export type PageStateEntry = { state: PageState; startTime: RelativeTime }
 export interface PageStateHistory {
   findAll: (startTime: RelativeTime, duration: Duration) => PageStateServerEntry[] | undefined
   isInActivePageStateAt: (startTime: RelativeTime) => boolean
+  wasInPageStateDuringPeriod: (state: PageState, startTime: RelativeTime, duration: Duration) => boolean
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏now that we have `wasInPageStateDuringPeriod` it could be nice to get rid of  `isInActivePageStateAt` wdyt?

Reply:
using `wasInPageStateDuringPeriod(ACTIVE, startTime, 0)` for those cases may not be super explicit with the `0` and the fact that we don't really care about the period there.

What about exposing:

```js
pageStateHistory = {
  wasInPageStateAt: (state: PageState, startTime: RelativeTime) =>
    pageStateHistory.wasInPageStateDuringPeriod(state, startTime, 0 as Duration),
  ...
}
```

Reply:
Looks good!